CSS/Homepage.css

```diff
@@ -16,13 +11,12 @@
 
 .section {
     margin: 50px 50px;
-    width: 600px;
-    height: 450px;
+    height: 280px;
```

Comment:
Avoid giving fixed values for section's height.

Reply:
change the heigth to a margin